test/e2e/dedicated_reader_test.go

```diff
@@ -0,0 +1,67 @@
+//+build e2e
+
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var _ = Describe("Openshift on Azure dedicated-cluster-reader e2e tests [DedicatedAdmin]", func() {
+	defer GinkgoRecover()
+
+	It("should read nodes", func() {
+		_, err := dreader.kc.CoreV1().Nodes().Get("master-000000", metav1.GetOptions{})
```

Comment:
check that you got a non-nil node object

Reply:
added